app/controllers/users_controller.rb

```diff
@@ -65,6 +65,8 @@ def create
   def create_external
     @user = User.new(params[:user])
     @user.password = generate_new_password
+    @user.created_by = current_user.try(:id)
```

Comment:
I'm fine ditching it.

If you do keep it, a few of our classes have `belongs_to :created_by_user, class_name: "User", foreign_key: :created_by` (which I think is awkward, but I think we did that to avoid changing any of the column names, which we don't like doing often because NU has a business analysis tool that sits on top of the DB and changing column names sometimes affects their queries.)